tests/yam/agama/auto.pm

```diff
@@ -7,16 +7,25 @@
 use base 'y2_installbase';
 use strict;
 use warnings;
+use serial_terminal;
 
 use testapi 'assert_screen';
 
 
 sub run {
     assert_screen('agama-main-page', 120);
     assert_screen('agama-installing', 60);
+    return 42;
 
     my @tags = ("welcome-to", "login");
     assert_screen \@tags, 960;
 }
 
+sub post_fail_hook {
+    my ($self) = shift;
+    $testapi::password = 'linux';
+    select_serial_terminal;
```

Comment:
there were some complains in progress ticket about not knowing what is the failure due to so many screens (but it is openQA design, no doubt), could you please check how would look with normal root console all this block here  https://openqa.opensuse.org/tests/3265003#step/auto/55 ?